### PR TITLE
Zomato api branch

### PR DESCRIPTION
**Pull request by sid-666**
I have just finished the main code on the zomato api functionalities.

### CODE REVIEW

**Review by Conanas — DISMISSED** [review]

**Review by Conanas — CHANGES_REQUESTED** [review]
update your tree with the master

File: Assets/script.js [hunk 1]
```diff
@@ -3,20 +3,17 @@ var currentWeather = {}
 
 // stores and returns the api key for the weather api
 function getWeatherApiKey() {
-    var weatherApiKey = "20ce152fba104603b4cb45bef144122a";
-    return weatherApiKey;
+    return "20ce152fba104603b4cb45bef144122a";
```
Comment (Conanas) [inline Assets/script.js]: change to 
var weatherApiKey = "20ce152fba104603b4cb45bef144122a";
return weatherApiKey;

File: Assets/script.js [hunk 2]
```diff
@@ -3,20 +3,17 @@ var currentWeather = {}
...
 }
 
 // retrieves and stores the latitude value for the responded weather data
 function getLatitude(response) {
-    var latitude = response.lat;
-    return latitude;
+    return response.lat;
```
Comment (Conanas) [inline Assets/script.js]: change to
var latitude = response.lat;
return latitude;

File: Assets/script.js [hunk 3]
```diff
@@ -3,20 +3,17 @@ var currentWeather = {}
...
 }
 
 // retrieves and stores the longitude value for the responded weather data
 function getLongitude(response) {
-    var longitude = response.lon;
-    return longitude;
+    return response.lon;
```
Comment (Conanas) [inline Assets/script.js]: change to 
var longitude = response.lon;
return longitude;

File: Assets/script.js [hunk 4]
```diff
@@ -51,6 +48,7 @@ function getFinishTime() {
     return finishTime;
 }
 
+// TODO - whys this here if its not doing anything
```
Comment (Conanas) [inline Assets/script.js]: remove this 
// TODO - whys this here if its not doing anything

File: Assets/script.js [hunk 5]
```diff
@@ -116,7 +114,7 @@ function findMinTemp(temps) {
     // sets the minimum temperature to the first element of the temperature arry
     var minTemp = temps[0];
     // for each element of the temperature arry
-    temps.forEach(function(temp) {
+    temps.forEach(function (temp) {
```
Comment (Conanas) [inline Assets/script.js]: change to
temps.forEach(function(temp) {

File: Assets/script.js [hunk 6]
```diff
@@ -133,7 +131,7 @@ function findMaxTemp(temps) {
     // sets the maximum temperature to the first element of the temperature arry
     var maxTemp = temps[0];
     // for each element in the temperature array
-    temps.forEach(function(temp) {
+    temps.forEach(function (temp) {
```
Comment (Conanas) [inline Assets/script.js]: change to
temps.forEach(function(temp) {

File: Assets/script.js [hunk 7]
```diff
@@ -152,7 +150,7 @@ function findAverageTemp(temps) {
     // variable to calculate the sum of the array
     var sum = 0;
     // for each temperature in the arry
-    temps.forEach(function(temp) {
+    temps.forEach(function (temp) {
```
Comment (Conanas) [inline Assets/script.js]: change to
temps.forEach(function(temp) {

File: Assets/script.js [hunk 8]
```diff
@@ -196,7 +194,7 @@ function processHourlyWeatherData(response) {
     currentWeather.temps = [];
 
     // go through every hour for the next 48 hours and display the data in the console
-    response.data.forEach(function(dataObject) {
+    response.data.forEach(function (dataObject) {
```
Comment (Conanas) [inline Assets/script.js]: change to
response.data.forEach(function(dataObject) {

File: Assets/script.js [hunk 9]
```diff
@@ -320,13 +318,13 @@ function callWeatherApi() {
     $.ajax({
         url: queryUrl,
         method: "GET"
-            // process retrieved weather data
+        // process retrieved weather data
```
Comment (Conanas) [inline Assets/script.js]: keep the new version here with

File: Assets/script.js [hunk 10]
```diff
@@ -320,13 +318,13 @@ function callWeatherApi() {
...
     }).
-    then(processHourlyWeatherData).
+        then(processHourlyWeatherData).
```
Comment (Conanas) [inline Assets/script.js]: remove whitespce by "back" tabbing this line

File: Assets/script.js [hunk 11]
```diff
@@ -320,13 +318,13 @@ function callWeatherApi() {
...
         // catch error if call is unsuccessful
-    catch(function(error) {
-        console.log("Catch error");
-    });
+        catch(function (error) {
+            console.log("Catch error");
+        });
```
Comment (Conanas) [inline Assets/script.js]: remove the whitespace by backtabbing these lines, remove whitespace between function and (error) to make function(error)

File: Assets/script.js [hunk 12]
```diff
@@ -412,4 +410,160 @@ function renderChosenWears() {
 
 // activates the call to the weather api
 callWeatherApi();
-// renderChosenWears();
```
Comment (Conanas) [inline Assets/script.js:415]: remove this comment

File: index.html [hunk 13]
```diff
@@ -9,106 +9,110 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
     <link rel="stylesheet" href="Assets/style.css">
 
-    <body>
+<body>
```
Comment (Conanas) [inline index.html:12]: keep the new version here

File: index.html [hunk 14]
```diff
@@ -9,106 +9,110 @@
...
 
 
-        <div class="header">
-            <button class="btn btn1 waves-effect">Wear</button>
-            <button class="btn btn1 waves-effect">Eat</button>
-        </div>
```
Comment (Conanas) [inline index.html:18]: keep the new version here

File: index.html [hunk 15]
```diff
@@ -9,106 +9,110 @@
...
 
-        <div class="container">
-            <form id="location-form">
-                <input class="form-control" id="location" type="text" placeholder="Location">
-            </form>
-            <form id="Country-form">
-                <input class="form-control" id="country" type="text" placeholder="Country">
-            </form>
-        </div>
+    <div class="container">
```
Comment (Conanas) [inline index.html:20]: keep the new version here

File: index.html [hunk 16]
```diff
@@ -9,106 +9,110 @@
...
+            </select>
+        </div>
+        <div id="confirmBtn">
+            <button class="btn confirmBtn" type="submit">Confirm</button>
+        </div>
+        <button id="eatform">eatsubmit</button>
+        <div class="weatherDisplay">
+        </div>
+        <div class="restRecommend">
+        </div>
```
Comment (Conanas) [inline index.html:102]: keep your version here

File: index.html [hunk 17]
```diff
@@ -9,106 +9,110 @@
...
 
 
-        </div>
+    </div>
```
Comment (Conanas) [inline index.html:105]: keep your version here

File: index.html [hunk 18]
```diff
@@ -9,106 +9,110 @@
...
 
-        <!-- moment.js -->
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.28.0/moment-with-locales.js" integrity="sha512-2Zf8MmDD4CaXNATmBqBXxW+D627jYF3yJNbZGBPmer+VYoEzqEkUHkz4xV9qcCjFdCxueRDd5/Gs8k6cGrWWXA==" crossorigin="anonymous"></script>
+    <!-- moment.js -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.28.0/moment-with-locales.js"
+        integrity="sha512-2Zf8MmDD4CaXNATmBqBXxW+D627jYF3yJNbZGBPmer+VYoEzqEkUHkz4xV9qcCjFdCxueRDd5/Gs8k6cGrWWXA=="
+        crossorigin="anonymous"></script>
```
Comment (Conanas) [inline index.html:110]: keep your version here

File: index.html [hunk 19]
```diff
@@ -9,106 +9,110 @@
...
 
-        <!-- jQuery -->
-        <script src="https://code.jquery.com/jquery-3.5.1.js" integrity="sha256-QWo7LDvxbWT2tbbQ97B53yJnYU3WhH/C8ycbRAkjPDc=" crossorigin="anonymous"></script>
-        <script src="Assets/script.js"></script>
-    </body>
+    <!-- jQuery -->
+    <script src="https://code.jquery.com/jquery-3.5.1.js"
+        integrity="sha256-QWo7LDvxbWT2tbbQ97B53yJnYU3WhH/C8ycbRAkjPDc=" crossorigin="anonymous"></script>
+    <script src="Assets/script.js"></script>
+</body>
```
Comment (Conanas) [inline index.html:116]: keep your version here

**Review by Conanas — COMMENTED** [review]
just need to update the lines

**Comment by sid-666** [comment]
I am making the changes required for easier mergin

**Review by Conanas — APPROVED** [review]
LGTM

**Pull request merged**